cardinal/ecs/storage/layout.go

```diff
@@ -31,6 +31,8 @@ func (l *Layout) Components() []component.IComponentType {
 }
 
 // HasComponent returns true if the ArchLayout has the given component type.
+//
+// Deprecated: Use component.Contains instead
 func (l *Layout) HasComponent(componentType component.IComponentType) bool {
```

Comment:
Are we gonna remove this later or is the plan to leave it in anyway?

Reply:
I plan on taking out the Layout struct entirely, so this will end up being taken out.